src/cli/build.ts

```diff
@@ -23,11 +23,40 @@ import {
   validImagePullSecret,
   generateYamlAndWriteToDisk,
 } from "./build.helpers";
+import { ModuleConfig } from "../lib/core/module";
 
 const peprTS = "pepr.ts";
 let outputDir: string = "dist";
 export type Reloader = (opts: BuildResult<BuildOptions>) => void | Promise<void>;
 
+type PeprConfig = {
+  pepr: Partial<ModuleConfig> & {
+    includedFiles: string[];
+    webhookTimeout?: number;
+    peprVersion: string;
+    uuid: string;
+  };
+  description: string;
+  version?:string;
+};
+
+type LoadModuleReturn = {
+  cfg: PeprConfig;
```

Comment:
```suggestion
  cfg: PeprConfig;
```

Pretty sure this is not not technically accurate, these are the [Pepr Config docs](https://docs.pepr.dev/main/user-guide/customization/#packagejson-configurations-table)

Reply:
Reviewed pepr docs against the implementation, it looks like we cover the relevant properties.